setup-cf-deploy-test-teardown.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+target="$1"
+secdir="${2:-../cloudfoundry/secure}"
+secfile="concourse-secrets.yml.gpg"
+secrets="${secdir}/${secfile}"
+
+# EV 'PP' = prefix to pipeline name for local customization of test
+#           pipelines
+
+if [ ! -f "${secrets}" ]
+then
+    echo -e 1>&2 "$0: Failed to find the secrets file ${secrets}\n\tPlease specify the correct directory holding \"${secfile}\"."
+    exit 1
+fi
+
+fly -t "$target" set-pipeline \
+    -p ${PP}cf-deploy-test-teardown \
```

Comment:
I really like this option for adding a prefix but I think it should have a more descriptive name.